workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardTable.tsx

```diff
@@ -19,54 +20,78 @@ export type AppProps = {
 };
 interface DashboardTableProps {
 	apps: AppProps[];
+	emptyStateMessage: {
+		description1: string,
+	description2: string,
+	title: string};
 }
 
-export function DashboardTable({apps}: DashboardTableProps) {
-	return (
-		<ClayTable borderless className="dashboard-table-container">
-			<ClayTable.Head>
-				<ClayTable.Cell headingCell>
-					<div className="dashboard-table-header-name">
-						<span className="dashboard-table-header-text">
-							Name
-						</span>
+export function DashboardTable({apps, emptyStateMessage}: DashboardTableProps) {
+	const {description1, description2, title} = emptyStateMessage;
+	if (!apps.length) {
```

Comment:
@alinedoleron Please, add a blank line between the declaration of `description1, description2, title` and their usage.